dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -570,6 +576,37 @@ export default defineComponent({
       this.onThisPageLinks = [...new Set(this.onThisPageLinksStart.concat(this.selectedFinancialServiceOptions))];
     },
 
+    /**
+     * Converts the kpisSection data to a request-friendly object
+     * @param kpiSectionsModel Kpi Section data
+     * @returns ObjectType
```

Comment:
Should also rather describe what you can use it for than the datatype

Reply:
Done.